e2e/update_pnpm_lock/test.sh

```diff
@@ -54,8 +54,8 @@ ASPECT_RULES_JS_DISABLE_UPDATE_PNPM_LOCK=
 _sedi 's#"@types/node": "16"#"@types/node": "14"#' package.json
 
 # Trigger the update of the pnpm lockfile
-if ! bazel run "$BZLMOD_FLAG" @npm//:sync; then
-    echo "ERROR: expected 'bazel run $BZLMOD_FLAG @npm//:sync' to pass"
+if bazel run "$BZLMOD_FLAG" @npm//:sync; then
```

Comment:
Maybe run it again to test that `bazel run` exits 0 on the 2nd run?

Reply:
It runs `bazel test //...` below and asserts 0 which is similar...? The mix of `bazel test` and `bazel run` might be the reason I previously had switched this to `bazel test`...

Reply:
Similar but I think its still worth asserting that `bazel run @npm//:sync` exits 0 on the 2nd go

Reply:
Defence in-depth